libminifi/include/core/ConfigurableComponent.h

```diff
@@ -212,6 +212,8 @@ class ConfigurableComponent {
   // Dynamic properties
   std::map<std::string, Property> dynamic_properties_;
 
+  virtual Property* findProperty(const std::string& name) const;
+
```

Comment:
Doesn't getProperty already handle this use case?

Reply:
In the python processors' properties are stored separately in the `python_properties_` vector. Because of this we need to override how to search for a property in getProperty and setProperty functions in the `ExecutePythonProcessor` class. Overriding this `findProperty` provides this, all other `getProperty` methods are either template functions or have an output reference parameter, which we cannot use for this.

Reply:
I was under the impression that `getProperty` was already `virtual`, but apparently that's not the case, so this new customization point makes sense knowing this. Thanks!